### ISSUE TITLE
Automate Docker build and testing via Travis

### ISSUE THREAD

**Issue by bnaul** [cesium-ml/cesium_web#169]
I'd suggest just using `cesium_app/tests/frontend/test_pipeline_sequentially.py` for testing because it doesn't hit the DB directly but does test all of the major functionality.

- If master changes and the Travis build succeeds, build a new image
- Run `cesium_app/tests/frontend/test_pipeline_sequentially.py`
- If it passes, push the image to docker hub

**Comment by cesium-sync** [comment]
➤ H R commented:
Should this be run everytime any code is modified? Should it run in parallel to the current Travis build or should it be incorporated in the current build?

**Comment by cesium-sync** [comment]
➤ Brett Naul commented:
@stefanvdwalt what do you think? would this be an `after_success` or a separate task via the dependent build setup?

**Comment by stefanv** [comment]
I think `after_success` is fine for now; we can always factor it out later.  This way, at least we'll have a continual deployment option of sorts.

**Comment by cesium-sync** [comment]
➤ H R commented:
@stefanvdwalt @brettnaul1 What happens if the docker build fails, should the Travis build fail too? That would alert whoever did the PR that they did something that requires code change for the docker image too. If that's the case then the `after_success` is not a good option because "The exit code of `after_success`, `after_failure`, `after_script` and subsequent stages do not affect the build result." (Travis doc)

**Comment by bnaul** [comment]
IMO the best option for that would be to keep the main build green but have
some other kind of notification. @stefanv thoughts?

**Comment by cesium-sync** [comment]
➤ Brett Naul commented:
@hrx427 I made a `cesium-ml/docker_builder` repo that you can fork. When do you think you'll have a build script working?

**Comment by cesium-sync** [comment]
➤ H R commented:
@brettnaul1 It should be done by the end of the week

**Comment by stefanv** [comment]
Can we aim to have this done by end of the Wednesday working meeting, please?  At least an initial working version.

**Comment by cesium-sync** [comment]
➤ H R commented:
:thumbsup:
